src/Playwright/Transport/Connection.cs

```diff
@@ -372,25 +372,7 @@ private Exception CreateException(PlaywrightServerError error)
                 return new TimeoutException(error.Message);
             }
 
-            if (error.Message.Contains("Browser closed") || error.Message.Contains("Target closed") || error.Message.Contains("The page has been closed."))
```

Comment:
Why is it changing?

Reply:
It was like logic which did not have any effect. Drive-by cleanup.